src/test/java/org/openstreetmap/atlas/utilities/collections/OptionalIterableTest.java

```diff
@@ -31,6 +31,8 @@ public void iterationTest()
         Assert.assertTrue(iterator.hasNext());
         Assert.assertEquals(Integer.valueOf(1), iterator.next());
```

Comment:
Since we're updating tests, do you mind changing `optIt` here to be non-abbreviated to follow general repo conventions? :)

Reply:
Done